vmdb/spec/models/metric/ci_mixin/capture_spec.rb

```diff
@@ -0,0 +1,187 @@
+require "spec_helper"
+
+describe Metric::CiMixin::Capture do
+  require File.expand_path(File.join(File.dirname(__FILE__),
+                                     %w(.. .. .. tools openstack_data openstack_data_test_helper)))
+
+  before :each do
+    MiqRegion.seed
+    _guid, _server, @zone = EvmSpecHelper.create_guid_miq_server_zone
+
+    @mock_meter_list = OpenstackMeterListData.new
+    @mock_stats_data = OpenstackMetricStatsData.new
+
+    @ems_openstack = FactoryGirl.create(:ems_openstack, :zone => @zone)
+    @ems_openstack.stub(:list_meters).and_return(
+        OpenstackApiResult.new((@mock_meter_list.list_meters("resource_counters") +
+                                @mock_meter_list.list_meters("metadata_counters"))))
+
+    @vm = FactoryGirl.create(:vm_perf_openstack, :ext_management_system => @ems_openstack)
+    @vm.stub(:perf_init_openstack).and_return(@ems_openstack)
+  end
+
+  before do
+    @orig_log = $log
+    $log = double.as_null_object
+  end
+
+  after do
+    $log = @orig_log
+  end
+
+  def expected_stats_period_start
+    parse_datetime('2013-08-28T11:01:20Z')
+  end
+
+  def expected_stats_period_end
+    parse_datetime('2013-08-28T12:41:40Z')
+  end
+
+  context "2 collection periods total, end of 1. period has incomplete stat" do
```

Comment:
changed to 1., as I did in other tests. I've found it more readable when there are more tests on output